src/serializableStateInvariantMiddleware.ts

```diff
@@ -1,6 +1,14 @@
 import isPlainObject from './isPlainObject'
-
-export function isPlain(val) {
+import { Middleware } from 'redux'
+
+/**
+ * Returns true if the passed value is "plain", i.e. a value that is either
+ * directly JSON-serializable (boolean, number, string, array, plain object)
+ * or `undefined`.
+ *
+ * @param val The value to check.
+ */
+export function isPlain(val: any) {
```

Comment:
`isPlain(val: unknown): val is any` 😓

With negation types this probably would be `: val is not symbol`, but those are only in TS 3.4 or 3.5.

Reply:
can't we use `isPlain(val: unknown): val is undefined
  | string
  | boolean
  | number
  | any[]
  | null
  | { [x: string]: any }`

for now?

Reply:
To be honest, I think `isPlain(val: any): boolean` is totally fine. I don't think these more complex types offer any practical benefit.